plover/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+add_custom_command (
+  OUTPUT "${PROJECT_SOURCE_DIR}/plover/plover.c" "${PROJECT_SOURCE_DIR}/plover/plover.h"
```

Comment:
maybe the output at least should be moved into `src/`? or at least out the way of the haskell source?